tests/test_s3_bucket_policy.py

```diff
@@ -1,44 +0,0 @@
-"""
```

Comment:
There are quite a lot of deleted tests. Could you explain why?

Reply:
these tests are moved into another file